RulesInventory.csv

```diff
@@ -267,14 +267,8 @@ CA3072,UseXmlReaderForDataTableReadXmlSchema,Desktop,#N/A,Use XmlReader for Data
 CA3073,ReviewTrustedXsltUse,System.Xml,#N/A,Review Trusted Xslt use,"Review code to insure that System.Xml.Xsl.XsltSettings::TrustedXslt is set from a trusted source. This pattern allows processing script within XSL, which, on untrusted XSL input, may lead to malicious code execution.",No,,,,,,,,,,,
 CA3074,ReviewClassesDerivedFromXmlTextReader,System.Xml,#N/A,Review classes derived from XmlTextReader,"Review code to insure that  DTD prosessing is disabled on all instances of classes derived from System.Xml.XmlTextReader. Enabling DTD processing on the XML reader and using UrlResolver for resolving external XML entities may lead to information disclosure. Content from file system or network shares for the machine processing the XML can be exposed to attacker. In addition, an attacker can use this as a DoS vector.",No,,,,,,,,,,,
 CA5122,PInvokesShouldNotBeSafeCriticalFxCopRule,System.Runtime,#N/A,#N/A,#N/A,No,,,,,,,Low,,,,
-CA5350,MD5CannotBeUsed,System.Security.Cryptography.Algorithms,#N/A,Do not use MD5,MD5 is banned by SDL,No,,,,,,,,,,,
```

Comment:
Instead of deleteing these lines, you should mark the Port column as "No" and in the notes column mention that this has been merged into a new rule. If you do that, Larry's tool will close the bug that's already been filed for this with that note.